### PR TITLE
feat(prqlc): add comments for some prqlc compile options

### PR DESCRIPTION
**Pull request by eitsupi**
Related to #2954

### CODE REVIEW

File: prql-compiler/prqlc/src/cli.rs [hunk 1]
```diff
@@ -116,8 +116,12 @@ enum Command {
     SQLCompile {
         #[command(flatten)]
         io_args: IoArgs,
+
+        /// With this option set, the output SQL does not have a signature comment at the bottom
```
Comment (max-sixty) [inline prql-compiler/prqlc/src/cli.rs]:
```suggestion
        /// Exclude the signature comment containing the PRQL version
```

(no strong view, feel free to discard my suggestion)

**Review by max-sixty — APPROVED** [review]
Thanks!

File: prql-compiler/prqlc/tests/test.rs [hunk 2]
```diff
@@ -112,13 +112,13 @@ fn compile_help() {
 
     -t, --target <TARGET>
             Target to compile to
-
```
Comment (eitsupi) [inline prql-compiler/prqlc/tests/test.rs]: This change from pre-commit-ci seems made the test fail on the main branch.
https://github.com/PRQL/prql/actions/runs/5473711576/jobs/9967543314

@max-sixty Any idea to disable this?

Comment (max-sixty) [reply prql-compiler/prqlc/tests/test.rs]: Sorry, I guess I missed this on vacation. Thanks for the follow-up to fix...

**Pull request merged**